### PR TITLE
Update adopters.yaml for Adoptors page

### LINKED ISSUE #5643: Adopters page: some links have empty link text
See https://opentelemetry.io/ecosystem/adopters/, e.g.:

> image

Note that the "Learn more" link text for `Bureau.id` is empty. By chance we're generating an external link icon, otherwise, the link wouldn't appear at all.

### PR DESCRIPTION
**Pull request by Yuvraj-Achrekar**
Fix #5643 
Added reference title text for links in adopters page

### CODE REVIEW

File: data/ecosystem/adopters.yaml [hunk 1]
```diff
@@ -127,6 +129,7 @@
   url: https://www.vtex.com/
   components: [Collector, Go, JavaScript, .NET]
   reference: 'https://colocatedeventseu2023.sched.com/event/1Jo8E/ingesting-65-tb-of-telemetry-data-daily-through-open-telemetry-protocol-and-collectors-gustavo-pantuza-vtex'
+  referenceTitle: blog post
```
Comment (svrnm) [inline data/ecosystem/adopters.yaml:132]: that's not a blog post, I would say "presentation" or "talk" or similar

Comment (Yuvraj-Achrekar) [reply data/ecosystem/adopters.yaml:132]: should i refer it as "Event post"?

Comment (svrnm) [reply data/ecosystem/adopters.yaml:132]: I think "presentation" is the best title, since we have this for another example already

Comment (Yuvraj-Achrekar) [reply data/ecosystem/adopters.yaml:132]: ok i'll correct that as presentation

**Review by cartermp — APPROVED** [review]

**Review by svrnm — CHANGES_REQUESTED** [review]
There is something not right in the automatted formatting commit (https://github.com/open-telemetry/opentelemetry.io/pull/5662/commits/e60e6ed5962b30bb0b445d5f895e00e833c0d02a), I will take a look.

@Yuvraj-Achrekar that's nothing you need to worry about, as it is unrelated to your work, I just block this PR with "request changes" until this is resolved

**Review by svrnm — APPROVED** [review]

**Pull request merged**